scripts/add_elevation_to_network.py

```diff
@@ -35,6 +37,16 @@
                             help='Output directory for the updated network',
                             required=True)
 
+    arg_parser.add_argument('-we',
+                            '--write_elevation_to_network',
+                            help='Whether node elevation data should be written as attribute to the network; defaults to True',
+                            default=True)
+
+    arg_parser.add_argument('-ws',
+                            '--write_slope_to_network',
+                            help='Whether link slope data should be written as attribute to the network; defaults to True',
+                            default=True)
+
     arg_parser.add_argument('-sj',
                             '--save_jsons',
                             help='Whether elevation and slope dictionaries and report are saved; defaults to True',
```

Comment:
let's add `type=bool` here, also description says 'defaults to True' but it doesn't

Reply:
it has `default=True` on the next line!

Reply:
hah! that bit was collapsed, completely missed it, sorry! :D